source/Wormhole.cpp

```diff
@@ -88,6 +89,8 @@ void Wormhole::Load(const DataNode &node)
 		}
 		else if(key == "mappable")
 			mappable = !remove;
+		else if(key == "colored")
```

Comment:
Do we need this? Currently it enables/disables the link color, but if you want the default link color just don't provide a color and if you do why do you need to specify it here again?

Reply:
Oh I'm dumb